README.md

```diff
@@ -1,5 +1,7 @@
 # VDRS
-Cloud Elements has created a community of custom made VDRS that can be viewed, downloaded, and shared. This is an independent catalog of vdrs created by customers, partners and Cloud Elements developers. Additionally, this community supports our partners leveraging CloudElements technology.
+Cloud Elements has created a community of custom made [VDRs](https://docs.cloud-elements.com/home/common-resources-overview) that can be viewed, downloaded, and shared. This is an independent catalog of VDRs created by customers, partners and Cloud Elements developers. Additionally, this community supports our partners leveraging Cloud Elements technology.
 
 ## Contributing
-To contribute we ask that you create a folder for the VDR, using pascal casing for the folder name (ex. MyContacts). The folder should contain the VDR JSON as well as a short README file with a description of what you built.
\ No newline at end of file
+To contribute we ask that you create a folder for the VDR, using pascal casing for the folder name (ex. MyContacts). The folder should contain the VDR JSON as well as a short README file with a description of what you built.
+
+All VDRs contributed by Cloud Elements will follow the same directory structure for compatability between differing VDR engine versions and Cloud Element's cli tool, [the doctor](https://www.npmjs.com/package/ce-util/v/2.2.5).
```

Comment:
@kiawnna I would remove the version here to make this future proof.

Reply:
@nick-bair done!